client/client-multi/client-java-core/src/main/java/org/kaaproject/kaa/client/Kaa.java

```diff
@@ -43,14 +42,28 @@ public static KaaClient newClient(KaaClientPlatformContext context) throws KaaRu
     }
 
     public static KaaClient newClient(KaaClientPlatformContext context, KaaClientStateListener listener) throws KaaRuntimeException {
+        return newClient(context, listener, false);
+    }
+
+    /**
+     * Creating new Kaa client
+     *
+     * @param context           - Kaa platform specific context for client init
+     * @param listener          - notifier for changing client state
+     * @param isUserKeyStrategy - user strategy about creating public/private keys. In default case use previously created
```

Comment:
The name <b>isUserKeyStrategy</b> is confusing. I think it should be named <b>useAutogeneratedKeys</b>. Please add to the description meaning of its values. 'false' - use existing pre-generated public/private key pair (default behaviour). 'true' - auto-generate new public/private key pair if absent.